hallebarde-frontend/src/components/Download.jsx

```diff
@@ -25,7 +25,7 @@ export default class Download extends Component {
       })
       .then(({ data }) => {
         this.setState({'isLoading': false})
-        window.open(data)
+        window.open(data, '_self')
```

Comment:
What does `_self` do ? does it open a new tab ?

Reply:
Nope, it does the opposite. It opens in the same window, which helps us get rid of the popup authorization problems.